install.sh

```diff
@@ -75,6 +75,8 @@ install_nvm_from_git() {
       exit 1
     }
   else
+    echo "=> Cleanup old nvm dir '$INSTALL_DIR'"
+    command rm -rf $INSTALL_DIR/* $INSTALL_DIR/.*
```

Comment:
That's a lot of work - it would include all aliases, the "versions" folder, the upcoming cache folder, and all the pre-0.12 versions that are installed in the root.

If the issue is that installation is complaining about an existing directory, let's change that so it's not a problem, rather than removing the directory.